src/static/current-state/components/QueueItem.tsx

```diff
@@ -150,22 +153,24 @@ export class QueueItem extends React.Component<QueueItemProps> {
   };
 
   render() {
-    const { bitbucketBaseUrl, request, status } = this.props;
+    const {
+      bitbucketBaseUrl,
+      request: { buildId, pullRequestId, pullRequest, created },
+      status,
+    } = this.props;
 
     if (!status) return null;
 
     return (
       <a
         className={`${queueItemStyles} queue-item`}
-        href={request.buildId ? buildUrlFromId(bitbucketBaseUrl, request.buildId) : '#'}
+        href={buildId ? buildUrlFromId(bitbucketBaseUrl, buildId) : '#'}
       >
         <ak-grid layout="fluid">
-          <ak-grid-column size="10">
+          <ak-grid-column size={status.state === 'queued' ? 11 : 12}>
```

Comment:
What was your reasoning for this change?

Reply:
The columns allow a button to be positioned in the right column of the queue item. However, the remove button is only available on items that are 'queued', so I've allowed items with other statuses that won't have the button to use the full width for their content.